src/main/kotlin/calculator/Parser.kt

```diff
@@ -0,0 +1,28 @@
+package calculator
+
+import calculator.common.isNotNumber
+import calculator.common.isNotOperator
+
+class Parser(private val limit: Int) {
+
+    fun parse(input: String?): List<String> {
+        if (input.isNullOrBlank()) {
+            return emptyList()
+        }
+        return input.split(" ")
+    }
```

Comment:
- 코틀린은 언어 레벨에서 non-nullable 을 지원하기 때문에, `?` 키워드를 생략해주신다면 null-safe 한 코드 작성이 가능합니다.

```suggestion
    fun parse(input: String): List<String> {
        if (input.isBlank()) {
            return emptyList()
        }
        return input.split(" ")
    }
```

Reply:
파라미터를 non null 처리하고, 이와 관련해서 아규먼트도 처리하도록 수정하여 반영합니다. 😄